WRFotron2.0_CEMAC_ARC4/main_restart.bash

```diff
@@ -59,7 +59,7 @@ done
 msg "chem"
 
```

Comment:
This file will also need the changes that were applied to main.bash.

Reply:
That was silly of me to forget!